Makefile

```diff
@@ -105,8 +105,10 @@ cmakelint:
 		--step 'Run cmakelint'
 
 clang_tidy:
-	echo "clang-tidy local lint is not yet implemented"
-	exit 1
+	@$(PYTHON) tools/linter/clang_tidy \
```

Comment:
This can call the local runner instead which can encode all the args like `--parallel` and the `--clang-tidy-exe`